anime/views.py

```diff
@@ -62,24 +62,29 @@ def update_anime(id: int):
         anime_schema = AnimeSchema(partial=True)
 
     try:
-        schema_dict = anime_schema.loads(request.data)
+        entry_dict = anime_schema.loads(request.data)
     except ValidationError as e:
         return {"error": e.messages}, 400
 
-    _x = db.session.query(Anime).filter(Anime.anime_id == id).update(
-        {**schema_dict}
-    )
+    # Function to convert ORM obj to dictionary
+    row2dict = lambda r: {
```

Comment:
I think there is a more elegant way.
https://stackoverflow.com/questions/1958219/how-to-convert-sqlalchemy-row-object-to-a-python-dict

Reply:
I have tried it but does not work, plus many methods but nothing is working. The main task is to convert  sqlalchemy.engine.row.Row data class to dict., only the method with green tick in the link I shared works-which is what I have implemented.

Reply:
I think the issue solved. Please have a look on commit [7e7efc7](https://github.com/EsamSharaf/anime_API/pull/8/commits/7e7efc7cf43ad8046667e78f985880718c91dbd4)